tensorrt_llm/serve/openai_server.py

```
@@ -770,6 +798,141 @@ def register_mm_encoder_routes(self):
+                        array.array("f", embedding).tobytes()).decode("utf-8")
+                data.append(
+                    EmbeddingResponseData(index=idx, embedding=embedding))
+
+            num_prompt_tokens = sum(len(t) for t in token_ids_list)
+            usage = EmbeddingUsageInfo(prompt_tokens=num_prompt_tokens,
+                                       total_tokens=num_prompt_tokens)
+            response = EmbeddingResponse(model=request.model,
+                                         data=data,
+                                         usage=usage)
+            return JSONResponse(content=response.model_dump())
+        except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)